src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlNodeList.cs

```diff
@@ -31,7 +31,7 @@ public override int Count
         }
 
         // IList methods
-        public int Add(object value)
+        public int Add(object? value)
```

Comment:
Is `null` ever gets added into this list? If no, it makes sense to assume it cannot contain `null`s, and annotate is accordingly (since this collection is internal). This will help reduce the number of `!`s in code that uses it

Reply:
Oh, it implements `IList`. Missed that. Than there's no way around it I suppose..